discussion-questions.md

```diff
@@ -1,8 +1,17 @@
 #  How much time did you spend on this assignment?
 
-_replace this text with your answer_
+For the assignment, I spent about 6 hours in total. But I spent more than 2 hours on setting up the IDE.
 
 # Discussion questions / topics
 Below, add one or two things that you might want to discuss when we meet again.
 
-_replace this text with your discussion questions / topics_
\ No newline at end of file
+1. I am not sure what to do with scala return function. The function seems to be like this:
+def functionName ([parameters]) : [return type] = {
+   function body
+   return [expr]
+}
```

Comment:
The `return` is not usually needed. In Scala, we often just write the expression, and the function results in whatever expression comes last in the function body.